src/main/java/com/almightyalpaca/discord/jdabutler/commands/Command.java

```diff
@@ -0,0 +1,17 @@
+package com.almightyalpaca.discord.jdabutler.commands;
+
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+
+public interface Command {
+    void dispatch(String[] args, User sender, TextChannel channel, Message message, String content);
```

Comment:
Why not use the `GuildMessageReceivedEvent` instead of splitting into the individual entities,
I'm not sure what this achieves